gym_sapientino/assets/default_map.txt

```diff
@@ -0,0 +1,5 @@
+P bB g |
+ bp G r|
+G   pg |
+ rpG PB|
+rP Bg b|
```

Comment:
Very handy!
Is there a documentation about this format? (even a two lines docstring in some class)

Reply:
@cipollone 

https://github.com/whitemech/gym-sapientino/blob/e6f2bae06be06b19f23ae07fea0c2a5a37f06eef/docs/configure_map.ipynb

Reply:
very nice